### PR TITLE
Datasets graph

### PR DESCRIPTION
**Pull request by bbovenzi**
Improve UX for datasets by adding a dependency graph to the datasets page. Clicking on either the list to the left or the graph will highlight that dataset and show its details. Clicking on a dag will show a tooltip with a link to the dag's page.

Screen Shot 2022-08-13 at 12 22 54 PM

Screen Shot 2022-08-19 at 10 15 42 AM

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/www/package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "terser-webpack-plugin": "<5.0.0",
     "typescript": "^4.6.3",
     "url-loader": "4.1.0",
+    "web-worker": "^1.2.0",
```
Comment (ashb) [inline airflow/www/package.json:76]: Is this used anywhere?

Comment (bbovenzi) [reply airflow/www/package.json:76]: It's a peer dependency of `elk` which we use to calculate how to render the graph

**Review by ashb — APPROVED** [review]
LGTM other than the default position of a small graph.

(I haven't read the code in great detail, but have run it)

**Comment by ashb** [comment]
image

**Comment by bbovenzi** [comment]
It's a little hard to tell when the graph should be centered vs top-left aligned. So I added a center button at the bottom. In a later PR I'll probably have a "center on dataset" button too.

**Review by dstandish — APPROVED** [review]

**Pull request merged**